force-app/main/default/lwc/geFormField/geFormField.js

```diff
@@ -106,16 +107,7 @@ export default class GeFormField extends LightningElement {
      */
     @api
     isValid() {
-        // We need to check for invalid values, regardless if the field is required
-        let fieldIsValid = this.checkFieldValidity();
-
-        if (this.element !== null && this.element.required) {
-            return isNotEmpty(this.value)
-                && this.value !== this.CUSTOM_LABELS.commonLabelNone
-                && fieldIsValid;
-        }
-
-        return fieldIsValid;
+        return validityCheck(this);
```

Comment:
Is the isValid method used externally somewhere?  Wondering if it needs its @api tag.

Reply:
Yeah, unfortunately, this is still used in the formSection component.